scss/partials/_comments.scss

```diff
@@ -118,7 +118,7 @@ div.comments {
         font-size: 13px;
         color: rgba(78,90,107,0.8);
         font-weight: 300;
-        margin: 0 0 0 #{$comment-avatar-size + 16}px;
+        margin: 10px 0 10px;
```

Comment:
Just for later reference, you can use `10px 0;` here, no need to repeat the same at the end.
I'm sure you already know this but 
- `0px`: all 4 values
- `0px 1px`: top/bottom 0px left/right 1px
- `0px 1px 2px`: top 0px, left/right 1px, bottom 2px
- `0px 1px 2px 3px`: top 0px, right 1px, bottom 2px, left 3px